packages/components/src/stepper/check-icon.js

```diff
@@ -3,34 +3,38 @@
  */
 import { createElement } from '@wordpress/element';
 
-export default () => (
-	<svg
-		role="img"
-		aria-hidden="true"
-		focusable="false"
-		width="18"
-		height="18"
-		viewBox="0 0 18 18"
-		fill="none"
-		xmlns="http://www.w3.org/2000/svg"
-	>
-		<mask
-			id="mask0"
-			mask-type="alpha"
-			maskUnits="userSpaceOnUse"
-			x="2"
-			y="3"
-			width="14"
-			height="12"
+export default () => {
+	// we need a unique mask id because HTML ids are global in nature and collisions result in strange outcomes
+	const maskId = Math.floor( Math.random() * 10000000 );
```

Comment:
This looks good! Another approach I'm thinking of is using a global counter as 'markID' since we don't need to worry about threads issues. However, randomness should help in some conditions.

Reply:
I thought about that... but it seemed like a lot of extra complication using React context to manage this for an ID that will never be externally used. I'll add a prefix to the string though